### DISCUSSION TITLE
Results inference with Custom Model

### DISCUSSION THREAD

**Discussion by Baldabiou** [roboflow/notebooks#235] [Q&A]
When i do my inference, there is no "Results saved to..." at the end of the code execution. 
How do you display the path of the inference result in google collab?
![pbl collab](https://github.com/roboflow/notebooks/assets/142986480/58095584-76bf-4672-bf9f-a2068819fe43)

**Comment by StefanoGarzoni** [comment]
in the inference execution code line, did you put "Save=True" ? (if I'm not mistaken this is exactly what you need to add)

**Reply by Baldabiou** [reply]
Yes, thank you for your reply. I find the documentation of Utralytics yesterday and that's exactly what I needed.

**Reply by StefanoGarzoni** [reply]
no problem!
